src/Microsoft.AspNetCore.Blazor/Microsoft.AspNetCore.Blazor.csproj

```diff
@@ -1,7 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.JSInterop\Microsoft.JSInterop.csproj" />
+  </ItemGroup>
```

Comment:
It looks like the only reason for this reference is that ElementRef does custom serialization. Is that the case? Is this worth thinking harder about? (probably not) I can't imagine anyone will build a Blazor application without the interop layer.

Reply:
It's also to make the JSON utils available, as they are used in a few places in this project.

It might be that one day we can reference some other JSON utils from here and not reference `Microsoft.JSInterop`. For the time being I expect it's fine as-is.